validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #54.      p_values_found[radius] = p_value_found_a[0]
</pre><p>0 is the first precision &gt;= target?</p><p>if no values are found i assume the list is empty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
This is a numpy array of size 1.

For what it's worth, this is how these values were accessed previously in a different location in the file (under find_p_value())

To minimize the diff, I decided against refactoring it.